roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -13,5 +14,14 @@
   - reload systemd units
   - restart node
 
+- block:
+  - name: Remove existing systemd service
```

Comment:
I'm not sure if this is what we want?

If the node is already a system container (which was possible previously in all types of deploys), and we remove this file and the host was already updated (IE, this is a no-op because this host was already upgraded) I'm not sure that the service unit will be placed again via the atomic command.

Reply:
>If the node is already a system container (which was possible previously in all types of deploys)

On Atomic installs this was a different node service. As a result atomic install would crash and complain that service file already exists. Atomic install is not very smart though - it won't create a service file if its missing during a second run.

This part is WIP still

Reply:
Reworked this to remove the services only when node is not yet bootstrapped